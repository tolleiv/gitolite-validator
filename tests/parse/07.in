repo foo baz
    RW+                     =   alice @teamleads
    -   master              =   dilbert @devteam
    -   refs/tags/v[0-9]    =   dilbert @devteam
    RW+D dev/                =   dilbert @devteam
    RW                      =   dilbert @devteam
    R                       =   @managers
