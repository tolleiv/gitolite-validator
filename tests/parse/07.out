root:
  repo:
    paths:
      foo
      baz
    rules:
      rule:
        permission: RW+
        members:
          alice
          @teamleads
      rule:
        permission: -
        refex: master
        members:
          dilbert
          @devteam
      rule:
        permission: -
        refex: refs/tags/v[0-9]
        members:
          dilbert
          @devteam
      rule:
        permission: RW+D
        refex: dev/
        members:
          dilbert
          @devteam
      rule:
        permission: RW
        members:
          dilbert
          @devteam
      rule:
        permission: R
        members:
          @managers
